### ISSUE TITLE
Add Webpack (bundle) support

### ISSUE THREAD

**Issue by jsprw** [mikro-orm/mikro-orm#196]
**Is your feature request related to a problem? Please describe.**
Follow up of #194.

It would make distributing the application much easier if all dependencies are packed in a single file. Also it would make it possible to bundle it with Angular SSR.

**Describe the solution you'd like**
No directory scanning, preventing required imports as it would throw errors.

Don't know if entities will be succesfully imported but it looks like it if it is set up like:
https://github.com/mikro-orm/mikro-orm/issues/194#issuecomment-540790793

In Mikro-ORM it throws errors...
```
ERROR in ./node_modules/knex/lib/dialects/mssql/index.js
Module not found: Error: Can't resolve 'mssql' in 'C:\Users\XX\Repositories\mikro-orm-examples\node_modules\knex\lib\dialects\mssql'
resolve 'mssql' in 'C:\Users\XX\Repositories\mikro-orm-examples\node_modules\knex\lib\dialects\mssql'
  Parsed request is a module
  using description file: C:\Users\XX\Repositories\mikro-orm-examples\node_modules\knex\package.json (relative path: ./lib/dialects/mssql)
```

... while in TypeORM it shows warnings and successfully bundles the files

```
WARNING in ../node_modules/typeorm/platform/PlatformTools.js
Module not found: Error: Can't resolve 'ioredis' in '/builds/xx/xx/node_modules/typeorm/platform'
 @ ../node_modules/typeorm/platform/PlatformTools.js
 @ ../node_modules/typeorm/index.js
 @ ./api/core/database.ts
 @ ./api/index.ts
 @ ./server.ts
```

https://github.com/typeorm/typeorm/blob/master/src/platform/PlatformTools.ts#L31

**Additional context**
Example proj, read readme for setup instructions: https://github.com/jsprw/mikro-orm-webpack-test

**Comment by B4nan** [comment]
Have you seen this? https://github.com/nestjs/nest/issues/1706#issuecomment-474514484

Could you try those proposed solutions?

**Comment by jsprw** [comment]
Hi, 

I have been looking through the discussion and found a synthax to properly compile webpack by replacing all the unused (thus not installed packages) with node-nope (hacky but probably working fix). Most of these errors were caused by the knex dependency (https://github.com/knex/knex/issues/1128#issuecomment-517811600). 

Only mariadb/callback was an error caused by mikro-orm.

```
plugins: [
    ...
    new webpack.NormalModuleReplacementPlugin(/mssql|mysql(?!2)|postgres|pg|oracle(db)?|sqlite3|mariadb\/callback/, "node-noop"),
    ...
  ]
```

I have pushed a new version to the example project with this fix. I don't know if this fix works yet but I suspect it does as those excluded packages are not used by this project. Only mysql2 is used in that project.

Excluding all node modules also caused succesful compilation but it does not work as it requires a node_modules directory on the server and that defeats the purpose of bundling.
